test/hotspot/jtreg/runtime/8176717/TestInheritFD.java

```diff
@@ -41,6 +41,8 @@
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.util.Collection;
+import java.util.concurrent.TimeoutException;
```

Comment:
`import java.util.concurrent.TimeoutException; ` can be removed now I think. Otherwise I think it looks good, thanks for improving the test case.

Reply:
I decided to check the cause of the `CompletionException` using instanceof, so the import is needed.